src/common/legacy_config_opts.h

```diff
@@ -1434,7 +1434,7 @@ OPTION(rgw_curl_low_speed_limit, OPT_INT) // low speed limit for certain curl ca
 OPTION(rgw_curl_low_speed_time, OPT_INT) // low speed time for certain curl calls
 OPTION(rgw_copy_obj_progress, OPT_BOOL) // should dump progress during long copy operations?
 OPTION(rgw_copy_obj_progress_every_bytes, OPT_INT) // min bytes between copy progress output
-OPTION(rgw_copy_verify_object, OPT_BOOL) // verify if the copied object is identical to source
+OPTION(rgw_sync_obj_integrity, OPT_BOOL) // verify if the copied object from remote is identical to source
```

Comment:
as in earlier comment by casey, let's not use the word "integrity" for etag comparison

Reply:
@mattbenjamin - Does `rgw_sync_obj_etag_verify` sound better?
(BTW, `rgw_sync_obj_integrity` was chosen from one of @cbodley's suggestion)

Reply:
Sure, but yes:  I think rgw_sync_obj_etag_verify is ideal.

Reply:
Done. Added another commit.